podium-gateway/src/main/webapp/i18n/en/request.json

```diff
@@ -272,6 +272,12 @@
                     }
                 }
             }
+        },
+        "linkedRequest": {
+            "title": "Linked request",
+            "researcher": "Linked request message for researcher.",
```

Comment:
Define the messages as follows: 
Text on request details page for coordinator:

This is a linked request, the other organization(s) have also received a request, please contact the other collaborating organization(s) to discuss the handling of the request.


Text on request details page for researcher:

This is a linked request composed of different data types from different organizations from the same patient selection.